examples/nested/README.md

```diff
@@ -0,0 +1,10 @@
+# Possible Nesting Configurations
```

Comment:
Can you add a small blurb about using `namesapceOverride` to get the components into different namespaces?

A small pros/cons discussion of when to use the different approaches would be nice too.

Reply:
Was trying to take an iterative approach here. Getting solid examples of these configurations will take quite a bit of time to get right and document. But the diagrams are useful to show off what is currently possible with the chart. So, maybe 2  pr's?

Reply:
Yea thats fine.